app/protocol/flows/v4/blockrelay/ibd.go

```diff
@@ -590,6 +590,10 @@ func (flow *handleIBDFlow) resolveVirtual(estimatedVirtualDAAScoreTarget uint64)
 			if err != nil {
 				return err
 			}
+			percents := int(float64(virtualDAAScore-virtualDAAScoreStart) / float64(estimatedVirtualDAAScoreTarget-virtualDAAScoreStart) * 100)
+			if percents < 0 {
```

Comment:
How can this become negative? is it possible that `virtualDAAScore < virtualDAAScoreStart` ?
If it's possible that `estimatedVirtualDAAScoreTarget < virtualDAAScoreStart` then I would set `estimatedVirtualDAAScoreTarget = virtualDAAScoreStart` before the `for` loop

Reply:
Because in some edge cases `virtualDAAScoreStart` can be higher than `estimatedVirtualDAAScoreTarget`